06LessonSix/in-class-project/css/intel-style.css

```diff
@@ -1,35 +1,35 @@
 
 body {
-    font-family: 'Arial', sans-serif;
+    font-family: sans-serif;
 }
 
 header, main, footer {
     width: 1024px;
     margin: auto;
 }
 
-!header {
+header {
     background-color: #0071c5;
     padding: 10px;
 }
 
-header-#h1 {
+header, h1 {

```

Comment:
remember, the comma `,` means that in this case `header` AND `h1` share the same rules. It's that what we want?